src/test/java/com/spanish_inquisition/battleship/server/BattleshipServerTest.java

```diff
@@ -0,0 +1,86 @@
+package com.spanish_inquisition.battleship.server;
+
+import org.testng.Assert;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import java.io.IOException;
+import java.net.ServerSocket;
+import java.net.Socket;
+import java.util.List;
+
+import static com.spanish_inquisition.battleship.common.AppLogger.defaultLevel;
+import static com.spanish_inquisition.battleship.common.AppLogger.initializeLogger;
+import static com.spanish_inquisition.battleship.common.AppLogger.logger;
+
+public class BattleshipServerTest {
+    private ServerSocket serverSocket;
+    private final String SERVER_ADDRESS = "localhost";
+    private final int TEST_PORT = 6660;
+    private Socket client1;
+    private Socket client2;
+
+    @BeforeClass
+    public void beforeClass() {
+        initializeLogger();
```

Comment:
Logger in tests? Why? All output is in the console anyway when something goes wrong and when everything works well then why bother